config/database.yml.example

```diff
@@ -10,12 +10,11 @@ default: &default
 
 development:
  <<: *default
- database:
+ database: sample_project_development
 
 test:
  <<: *default
- database:
+ database: sample_project_test
 
 production:
- <<: *default
```

Comment:
shouldn't this be kept for general configs like adapter, encoding, pool, etc?

Reply:
I think those are managed on the URL itself or as config variables which Rails recognizes

Reply:
I think not everything is present in the URL, for instance: encoding, pool, any probably others